inference-engine/src/legacy_api/src/net_pass.cpp

```diff
@@ -304,6 +304,19 @@ void RemoveLayer(CNNLayerPtr& layer) {
 /****  Converter Passes  ************************************/
 /************************************************************/
 
+static RNNSequenceLayer::CellType cell_type_from_name(std::string& layer_type) {
```

Comment:
@DariaMityagina, this file does not belong to our plugin. Let us leave it without modification. It is unnecessary.